### DISCUSSION TITLE
[Bug]: Huggingface deployment instruction problem with space duplication

### DISCUSSION THREAD

**Discussion by alexlyzhov** [danny-avila/LibreChat#2707] [General]
### What happened?

The URL given https://huggingface.co/spaces/LibreChat/template in the instruction https://www.librechat.ai/docs/remote/huggingface says:
`This Space has been paused by its owner. Want to use this Space? Head to the [community tab](https://huggingface.co/spaces/LibreChat/template/discussions) to ask the author(s) to restart it.`
The "Duplicate this Space" button is instead located at the community tab on the URL: https://huggingface.co/spaces/LibreChat/template/discussions

### Steps to Reproduce

1. go to https://huggingface.co/spaces/LibreChat/template
2. click on URL https://www.librechat.ai/docs/remote/huggingface

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
The button is there, no need to go in the community tab:

![image](https://github.com/danny-avila/LibreChat/assets/32828263/39a98d4f-ed43-439f-97b3-106468bd9c7a)
